internal/lib/container_server.go

```diff
@@ -144,7 +144,7 @@ func New(ctx context.Context, configIface libconfig.Iface) (*ContainerServer, er
 }
 
 // LoadSandbox loads a sandbox from the disk into the sandbox store
-func (c *ContainerServer) LoadSandbox(id string) error {
+func (c *ContainerServer) LoadSandbox(id string) (err error) {
```

Comment:
Let's use `retErr` where we have defers relying on named errors.

Reply:
@mrunalp 
Comment has been addressed.